tools/validators/instance_validator/tests/telemetry_validator_test.py

```diff
@@ -18,14 +18,15 @@
 from __future__ import print_function
 
 import json
-# import threading
+from os import path
 
+from absl.testing import absltest
 from validate import entity_instance
 from validate import instance_parser
 from validate import telemetry_error
 from validate import telemetry_validator
-from absl.testing import absltest
-from os import path
+
+# import threading
```

Comment:
is this obsolete?  if so, remove.  Otherwise explain

Reply:
done